tests/componenttests/server/tests/mediaPipeline/SetSourcePositionTest.cpp

```diff
@@ -0,0 +1,327 @@
+ *   Expect that audio source is attached.
+ *   Attach the video source.
+ *   Expect that video source is attached.
+ *   Expect that rialto source is setup
+ *   Expect that all sources are attached.
+ *   Expect that the Playback state has changed to IDLE.
+ *
+ *  Step 4: Pause
+ *   Pause the content.
+ *   Expect that gstreamer pipeline is paused.
+ *
+ *  Step 5: Write 1 audio frame
+ *   Gstreamer Stub notifies, that it needs audio data
+ *   Expect that server notifies the client that it needs 3 frames of audio data.
+ *   Write 1 frame of audio data to the shared buffer.
+ *   Send HaveData message
+ *   Expect that server notifies the client that it needs 3 frames of audio data.
+ *
+ *  Step 6: Write 1 video frame
+ *   Gstreamer Stub notifies, that it needs video data
+ *   Expect that server notifies the client that it needs 3 frames of video data.
+ *   Write 1 frame of video data to the shared buffer.
+ *   Send HaveData message
+ *   Expect that server notifies the client that it needs 3 frames of video data.
+ *
+ *  Step 7: Notify buffered and Paused
+ *   Expect that server notifies the client that the Network state has changed to BUFFERED.
+ *   Gstreamer Stub notifies, that pipeline state is in PAUSED state
+ *   Expect that server notifies the client that the Network state has changed to PAUSED.
+ *
+ *  Step 8: Set Audio Source Position
+ *   Trigger set source position procedure for audio source
+ *   Expect that SetSourcePositionResponse has success status
+ *
+ *  Step 9: Set Video Source Position
+ *   Trigger set source position procedure for video source
+ *   Expect that SetSourcePositionResponse has success status
+ *
+ *  Step 10: End of audio stream
+ *   Send audio haveData with one frame and EOS status
+ *   Expect that Gstreamer is notified about end of stream
+ *
+ *  Step 11: End of video stream
+ *   Send video haveData with one frame and EOS status
+ *   Expect that Gstreamer is notified about end of stream
+ *
+ *  Step 12: Notify end of stream
+ *   Simulate, that gst_message_eos is received by Rialto Server
+ *   Expect that server notifies the client that the Network state has changed to END_OF_STREAM.
+ *
+ *  Step 13: Remove sources
+ *   Remove the audio source.
+ *   Expect that audio source is removed.
+ *   Remove the video source.
+ *   Expect that video source is removed.
+ *
+ *  Step 14: Stop
+ *   Stop the playback.
+ *   Expect that stop propagated to the gstreamer pipeline.
+ *   Expect that server notifies the client that the Playback state has changed to STOPPED.
+ *
+ *  Step 15: Destroy media session
+ *   Send DestroySessionRequest.
+ *   Expect that the session is destroyed on the server.
+ *
+ * Test Teardown:
+ *  Memory region created for the shared buffer is unmapped.
+ *  Server is terminated.
+ *
+ * Expected Results:
+ *  Set Source Position is handled and forwarded to the gstreamer.
+ *
+ * Code:
+ */
+TEST_F(SetSourcePositionTest, setAudioSourcePositionSuccess)
```

Comment:
Change name to just setSourcePositionSuccess

Reply:
changed.